### ISSUE TITLE
Support for Starfield

### ISSUE THREAD

**Issue by darmach** [Fr33dan/GPSaveConverter#64]
Is there a chance for adding Starfield support?

**Comment by szombi** [comment]
I don't think this tool currently supports this type of save game convert or I didn't find the proper wiki/issues. 
Someone in another tool explained how the Starfield saves works: https://github.com/Z1ni/XGP-save-extractor/blob/8d7faad67fc68259d188509ce4ea246d1b08b6b6/main.py#L289-L293 
Sorry, I don't want to advertise another tool I just found here a good description of the Starfield situation. 

Now I see the list of files, probably I can write regex for file translation but I couldn't figure out what we can do with the Blob ID.  The tool can find the pieces of the targeted `sfs` files but it should merge these files. 
![image](https://github.com/Fr33dan/GPSaveConverter/assets/60069531/969b057f-a92f-4c91-8294-3fab20d77977)
